src/app/clusters/closure-dimension-server/closure-dimension-matter-context.h

```diff
@@ -0,0 +1,65 @@
+/*
+ *
+ *    Copyright (c) 2024 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+ #pragma once
+
+ #include <app/data-model/Nullable.h>
+ #include <lib/core/CHIPError.h>
+ #include <lib/core/CHIPPersistentStorageDelegate.h>
+ #include <lib/core/DataModelTypes.h>
+ 
+ namespace chip {
+ namespace app {
+ namespace Clusters {
+ namespace ClosureDimension {
+ 
+ /** @brief
+  *    Interface to allow interaction with interaction model and ember layers. Can be faked for unit testing.
+  */
+ class MatterContext
+ {
+ public:
+     // TODO: remove touch on mEndpoint once this is used. I am apparently unable to locate the proper place to turn this off in the
+     // build file, so whatever, compiler, you win. I've touched it. You happy now?
```

Comment:
```suggestion
     // TODO: remove touch on mEndpoint once this is used. I am apparently unable to locate the proper place to turn this off in the
     // build file, so whatever, compiler, you win. I've touched it. You happy now?
```
you can clean this comment out

Reply:
done